### PR TITLE
Update index.html

### LINKED ISSUE #156: Change "Support" link text and destination
- [x] Change text to "Voting info"
- [x] Redirect hyperlink to the start of the "How to claim your vote" section

### LINKED ISSUE #157: Clarify "Preferred method of contact" on the rider form
Underneath "Preferred method of contact" heading, include text:
"How would you like to driver to get in touch with you?"

### PR DESCRIPTION
**Pull request by moinshaikh**
I have included the text : "How would you like to driver to get in touch with you?" underneath "Preferred method of contact" heading.

Thanks!

Note: This PR fixes #156 and fixes #157.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
                     <a href="#howtoclaim" class="nav__link scroll">How it works</a>
                 </li>
                 <li class="nav__item nav__support">
```
Comment (moinshaikh) [inline index.html]: This is done with regard to Issue #156

File: index.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+Preferred method of contact<!DOCTYPE html>
```
Comment (richardwestenra) [inline index.html]: Can you please remove this change? Looks like you might have accidentally pasted it at the start of the document?

**Comment by richardwestenra** [comment]
Note: This PR fixes #156 and fixed #157.

File: index.html (hunk 2 above)
Comment (moinshaikh) [inline index.html]: Sorry, my bad, yes I did it accidentally. I am removing and letting you know in couple of minutes.

Comment (richardwestenra) [inline index.html]: All good, thanks!

Comment (moinshaikh) [inline index.html]: Cool!

**Comment by richardwestenra** [comment]
LGMT. Merged, thanks!

**Pull request merged**